feature/gnmi/otg_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -106,53 +106,63 @@ func TestInterfaceCounters(t *testing.T) {
 	cases := []struct {
 		desc    string
 		path    string
-		counter *ygnmi.Value[uint64]
+		counter ygnmi.SingletonQuery[uint64]
+		skip    bool
 	}{{
 		desc:    "InUnicastPkts",
 		path:    intfCounterPath + "in-unicast-pkts",
-		counter: gnmi.Lookup(t, dut, intfCounters.InUnicastPkts().State()),
+		counter: intfCounters.InUnicastPkts().State(),
+	}, {
```

Comment:
why is this repeated twice?

Reply:
Also present in the ATE version of the test, I've now deleted it here.